.github/CODEOWNERS

```diff
@@ -15,8 +15,8 @@
 #   Please refer the link for the detail
 #   https://help.github.com/en/articles/about-code-owners
 
-*.c                 @moidx
-*.h                 @moidx
+*.c                 @lowRISC/ot-c-cpp-reviewers
+*.h                 @lowRISC/ot-c-cpp-reviewers
```

Comment:
These make the existing sw rules a bit redundant. I'm happy to send a follow up PR though.

Reply:
Done. Removed redundant sw rules.